Lesson33/Gokhan/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { List } from './components/List';
+import { Navbar } from './components/Navbar';
+import { Account } from './components/Account';
+import { PerformanceState } from './components/PerformanceState';
+import { LogIn } from './components/LogIn';
+import { UserProvider, useUserContext } from './components/Context/UserContext';
+import './App.css';
+
+const initialTodos = [
+	{ id: 1, text: 'Do not give up', completed: false },
+	{ id: 2, text: 'Keep Dreaming', completed: true },
+	{ id: 3, text: 'Follow Your Dreams', completed: false },
+	{ id: 4, text: 'Work Hard', completed: false },
+	{ id: 5, text: 'Think Possitive', completed: true },
+	{ id: 6, text: 'Finish Course', completed: false },
+];
+
+export const App = () => {
+	const [todos, setTodos] = useState(initialTodos);
+
+	return (
+		<UserProvider>
+			<div className='container'>
+				<Navbar />
+				<MainContent todos={todos} setTodos={setTodos} />
+			</div>
+		</UserProvider>
+	);
+};
+
+const MainContent = ({ todos, setTodos }) => {
+	const { state, dispatch } = useUserContext();
+
+	const updateUsername = (newUsername) => {
+		dispatch({ type: 'UPDATE_USERNAME', payload: { username: newUsername } });
+	};
+
+	return (
+		<>
+			{state.isLoggedInUser ? (
+				<>
+					<Account incomingUsername={state.username} updateUsername={updateUsername} />
```

Comment:
You don't need to pass props around when you use Context. The purpose of using Context API is to be able to access data everywhere without a need to pass props. Try to refresh the concept by watching this video: https://youtu.be/t9WmZFnE6Hg?si=hfgCnV-lkYmQPrlh

After, update Account component so it does not takes or uses props. You should be accessing data from Context directly inside Account component.